### ISSUE TITLE
[BUG] Accessibility issue with File Upload control

### ISSUE THREAD

**Issue by PurpleDuckMedia** [formio/formio.js#2258]
Hi currently using the file upload control as part of an angular js solution. However, we noticed that the file upload control is not as accessible as it could be. In particular, the file delete header is missing making it difficult for screen readers. So I was wondering if there was a way to override the layout of the builtin file upload control without having to code a custom control. In particular, adding a th to the delete and changing the order of the columns (moving delete to the end)

`<thead>
  <tr>
    
    
    <th>File Name</th>
    <th>Size</th>
    <th ng-if="!readOnly" style="width:1%;white-space:nowrap;" class="ngscope">Remove File</th>
  </tr>
</thead>`

![FileUpload Control](https://user-images.githubusercontent.com/28952958/72146835-3628ab00-3395-11ea-803f-316042e9708f.PNG)

**Comment by randallknutson** [comment]
If this is an accessibility issue, we'd love to have a fix in our core library. The template for this view is at https://github.com/formio/formio.js/blob/master/src/templates/bootstrap/file/form.ejs

If you wanted to make the changes and submit a pull request we'd love to have a fix to it. 

If not, you can override the template in your own application or website by using the template overrides. Documentation is at https://github.com/formio/formio.js/wiki/Templates-API

**Comment by jeriah-formio** [comment]
We're currently addressing a backlog of GitHub issues. Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
